bot.py

```diff
@@ -5,26 +5,69 @@
 
 client = commands.Bot(command_prefix = "$")
 
-conn = sqlite3.connect('users.db')
+conn = sqlite3.connect(':memory:')
 
 c = conn.cursor()
 
 c.execute("""CREATE TABLE IF NOT EXISTS users (
              irl,
-             discord
+             game,
+             username
              )""")
 
-@client.event
-async def on_ready():
-    print("Bot is ready.")
-    print("test")
+def start_bot():
+    @client.event
+    async def on_ready():
+        print('Bot is ready.')
 
-@client.event
-async def on_member_join(member):
-    print(f"{member} has joined the server")
+    @client.event
+    async def on_member_join(member):
+        print(f'{member} has joined the server')
 
-@client.event
-async def on_member_remove(member):
-    print(f"{member} has left the server")
+    @client.event
+    async def on_member_remove(member):
+        print(f'{member} has left the server')
 
-client.run(config.token)
\ No newline at end of file
+    @client.command()
+    async def dnsbot(ctx, flag = None, *args):
+        if flag == None:

```

Comment:
I think these first 2 if statements can be combined, because either way, the user will need help. As a bonus, you can remove a level of indentation:

```
if flag is None or flag[0] != '-': # my comment below about `flag` being non-empty still applies here
    await ctx.send('Type "dnsbot -h" for help')
    return

arg_num = len(args)
...
```